docker/Dockerfile_dev.cpu

```diff
@@ -20,12 +33,10 @@ RUN apt-get update && \
 RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 1
 RUN update-alternatives --install /usr/local/bin/pip pip /usr/local/bin/pip3 1
 
-RUN pip install --no-cache-dir psutil
-RUN pip install --no-cache-dir torch
-RUN pip install --no-cache-dir torchvision
-RUN pip install --no-cache-dir torchtext
+RUN pip install --no-cache-dir psutil future torch torchvision torchtext
 ADD serve serve
```

Comment:
Given that the docker build would be run from the serve/docker directory shouldn't this be
```suggestion
ADD ../serve serve
```